### ISSUE TITLE
Cannot install via npm

### ISSUE THREAD

**Issue by weierz** [xxhomey19/github-uploader#3]
```bash
npm install -g github-uploader
/usr/local/bin/github-uploader -> /usr/local/lib/node_modules/github-uploader/lib/cli.js
/usr/local/bin/gu -> /usr/local/lib/node_modules/github-uploader/lib/cli.js

> puppeteer@0.13.0 install /usr/local/lib/node_modules/github-uploader/node_modules/puppeteer
> node install.js

ERROR: Failed to download Chromium r515411! Set "PUPPETEER_SKIP_CHROMIUM_DOWNLOAD" env variable to skip download.
{ Error: read ETIMEDOUT
    at _errnoException (util.js:1024:11)
    at TLSWrap.onread (net.js:615:25) code: 'ETIMEDOUT', errno: 'ETIMEDOUT', syscall: 'read' }
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! puppeteer@0.13.0 install: `node install.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the puppeteer@0.13.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/willbeethoven/.npm/_logs/2017-12-18T05_24_19_789Z-debug.log

```

**Comment by homerchen19** [comment]
Hi @WillBeethoven ,

It seems to be network issue.
https://github.com/GoogleChrome/puppeteer/issues/1597

**Comment by weierz** [comment]
Well, I struggled with this issue for two hours this morning. After running `npm install -g cnpm --registry=https://registry.npm.taobao.org` and `cnpm i puppeteer`, I thought it should work but failed at login with the access error. So I downloaded `nvm` to [avoid writing access error](https://docs.npmjs.com/getting-started/fixing-npm-permissions#how-to-prevent-permissions-errors). Then running `cnpm install -g github-uploader`. 

Finally, it works.
